app/Commands/PullCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Commands;
+
+use App\Services\MiseService;
+use Illuminate\Support\Facades\Process;
+use Illuminate\Support\Facades\Storage;
+use LaravelZero\Framework\Commands\Command;
+
+use function Laravel\Prompts\confirm;
+use function Laravel\Prompts\error;
+use function Laravel\Prompts\info;
+use function Laravel\Prompts\multiselect;
+use function Laravel\Prompts\note;
+
+class PullCommand extends Command
+{
+    protected $signature =
+        'pull {recipe?*}' .
+        '{--no-process : prevent processes from executing}';
+
+    protected $description = 'Pull a recipe from mise.dev';
+
+    public function handle(): void
+    {
+        if ($this->option('no-process')) {
+            info('Dry run enabled');
+            Process::fake();
+        }
+
+        $remoteRecipes = $this->selectRemoteRecipes();
+
+        foreach ($remoteRecipes as $recipe) {
+            $this->install($recipe);
+        }
+
+        if (confirm('Do you want to run the recipe now?')) {
+            $this->call('apply', [
+                'recipe' => $remoteRecipes,
+                '--no-process' => $this->option('no-process'),
+            ]);
+        }
+    }
+
+    protected function selectRemoteRecipes(): array
+    {
+        $selectedRemoteRecipes = $this->argument('recipe');
+
+        if (empty($selectedRemoteRecipes)) {
+            return multiselect(
+                label: 'Which recipe(s) should I pull?',
+                options: app(MiseService::class)->allForSelect(),
```

Comment:
OK, so there are a few things I have questions about.

How do we handle:
- Recipes that we already have locally and haven't changed
- Recipes that we already have locally and have changed
- Recipes that we have a local one with the same name but it's not the same